wasm/src/programs/data/plaintext.rs

```diff
@@ -108,6 +110,12 @@ impl Plaintext {
         Ok(Uint8Array::from(rust_bytes.as_slice()))
     }
 
+    /// Get the left endian boolean array representation of the plaintext.
+    #[wasm_bindgen(js_name = "toBitsLe")]
+    pub fn to_bits_le(&self) -> Array {
+        to_bits_array_le!(self)
+    }
+
```

Comment:
We should create from_bits_le, from_fields, and to_fields on this methods and on the ciphertext object too.